l5/cypress/e2e/todo.cy.ts

```diff
@@ -160,33 +135,22 @@ describe("Level 5 milestone should validate protected route after signin", () =>
     cy.get("button[type='submit']").click();
   });
 
-  it("and redirect to Home page when authenticated", () => {
+  it("the application should redirect to the homepage and the `<nav>` element should be visible", () => {
     cy.get("nav").should("be.visible");
   });
 
-  it("and redirect to Signin page when clicked on Signout", () => {
+  it("the application should redirect to the `/signin` page if the user signs out", () => {
     cy.visit(studentSubmissionUrl + "/signin");
```

Comment:
Why is the user visiting the `/signin` page to _sign out_? The test description says that the user's action is signing out.

Reply:
Agree. This is a remnant test of a feature we already implemented in the lesson. Removed this and have updated a test for the Back button navigation later.